### PR TITLE
Added "gets or sets" rule for the CSharp model template

### PR DESCRIPTION
**Pull request by brnleehng**
@hovsepm pointed out there were files that were committed, but were not changed at all. He suggested to just create a new PR without those files.

@tbombach Made the changes for the method descriptions

#989 This is the reference for previous PR and the issue  is #981

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@brnleehng__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (brhoan). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

File: AutoRest/Generators/CSharp/CSharp/ClientModelExtensions.cs [hunk 1]
```diff
@@ -132,6 +132,36 @@ private static bool ShouldValidate(this IType model)
...
+        /// Format the documentation of a property properly with the correct getters and setters. Note that this validation will
+        /// checks for special cases such as acronyms and article words.
+        /// </summary>
+        /// <param name="property">The given property documentation to format</param>
+        /// <returns>A reference of the property documentation</returns>
+        public static string GetFormattedPropertyDocumentation(this Property property)
+        {
+            if (string.IsNullOrEmpty(property.Documentation))
+            {
+                return property.Documentation.EscapeXmlComment();
```
Comment (hovsepm) [inline AutoRest/Generators/CSharp/CSharp/ClientModelExtensions.cs]: if property.Documentation is null this code will have null ref exception.

**Comment by hovsepm** [comment]
:shipit:

**Pull request merged**